src/components/Proposal/Create/PluginForms/CreateKnownGenericPluginProposal.tsx

```diff
@@ -380,7 +383,7 @@ class CreateKnownPluginProposal extends React.Component<IProps, IState> {
                 }
 
                 if (field.type.includes("bytes")) {
-                  if (!isHexString(value)) {
+                  if (value && !isHexString(value)) {
```

Comment:
I think it may be wrong to expect this to be a hex string.  the data type is bytes.  I seem to recall the contract saying it ben be any data.  Are you sure this cannot in fact be *any* string value?  This is what you are implying in the placeholder text.

Reply:
it is hex string